app/services/hyrax/persist_directly_contained_output_file_service.rb

```diff
@@ -24,7 +24,7 @@ def self.call(content, directives)
     def self.retrieve_file_set(directives)
       uri = URI(directives.fetch(:url))
       raise ArgumentError, "#{uri} is not an http(s) uri" unless uri.is_a?(URI::HTTP)
-      ActiveFedora::Base.find(ActiveFedora::Base.uri_to_id(uri.to_s))
+      Valkyrie.config.metadata_adapter.query_service.find_by_alternate_identifier(alternate_identifier: ActiveFedora::Base.uri_to_id(uri.to_s), use_valkyrie: false)
```

Comment:
Do we need to cast the URI? Can we use `uri` instead of `ActiveFedora::Base.uri_to_id(uri.to_s)`?

Reply:
uri_to_id is being addressed by another issue so I left this as is.

Reply:
https://github.com/samvera/hyrax/issues/3821